src/NuGet.Core/NuGet.Build.Tasks.Pack/PackTask.cs

```diff
@@ -129,7 +129,7 @@ public override bool Execute()
             }
             catch (Exception ex)
             {
-                ExceptionUtilities.LogException(ex, Logger);
+                ExceptionUtilities.LogException(ex, Logger, true);
```

Comment:
What happens when a PackagingException is thrown?

In particular take this project: 

```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <IncludeBuildOutput>false</IncludeBuildOutput>
    <TargetFramework>net5.0</TargetFramework>
  </PropertyGroup>

</Project>
```

Today if you run it, it logs:

```
C:\Program Files\dotnet\sdk\5.0.200-preview.20601.7\Sdks\NuGet.Build.Tasks.Pack\build\NuGet.Build.Tasks.Pack.targets(221,5): error NU5017: Cannot create a package that has no dependencies nor content. [C:\Users\Nikolche\Source\Repos\PR3853\PR3853.csproj]
```